easybuild/easyconfigs/l/libevent/libevent-2.0.22-GCC-4.9.2.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.0.22'
+
+homepage = 'http://libevent.org/'
+description = """The libevent API provides a mechanism to execute a callback function when a specific
+ event occurs on a file descriptor or after a timeout has been reached.
+ Furthermore, libevent also support callbacks due to signals or regular timeouts."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+source_urls = ['https://github.com/downloads/libevent/libevent/']
+sources = ['%(name)s-%(version)s-stable.tar.gz']
+source_urls = ['https://sourceforge.net/projects/levent/files/%(name)s/%(name)s-2.0/']
```

Comment:
well, the 2nd overrides the 1st, so I guess the first one should be dropped?

also, use `%(version_major_minor)s` rather than hardcoding the `2.0`